docs/manifest/manifest.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Office Add-ins manifest reference
-description: The elements of an Office Add-ins XML manifest.
+title: Add-in only manifest reference for Office Add-ins
+description: The elements of an Office Add-ins add-in only manifest.
```

Comment:
Consider adding the format here to help with SEO (in case people search for information on the manifest XML). 
```suggestion
description: The elements of the XML-based Office Add-ins add-in only manifest.
```